kafka-mixin/alerts/alerts.libsonnet

```diff
@@ -63,21 +59,21 @@
           {
             alert: 'KafkaISRExpandRate',
             expr: |||
-              sum by(instance, kafka_cluster) (rate(kafka_server_ReplicaManager_IsrExpandsPerSec{%(kafkaFilteringSelector)s}[5m])) != 0
+              sum (rate(kafka_server_replicamanager_isrexpandspersec{%(kafkaFilteringSelector)s}[5m])) != 0
```

Comment:
sum() will likely drop all labels, so instance/kafka_cluster would not be available in descriptions.

Reply:
- if you do sum(<metric>) you would get just sum with all labels dropped
- if you do sum by (<labe1l>) (<metric>) you would get just sum by (<label1>), thus only <label1> would present in result, others dropped.
- if you do sum without (<label1>) (<metric>) you would get sum by all labels, except <label1>
- you also just use expression like  `<metric> > x` if you don't need any aggregation at all.

"
These operators can either be used to aggregate over all label dimensions or preserve distinct dimensions by including a without or by clause. These clauses may be used before or after the expression.

`<aggr-op> [without|by (<label list>)] ([parameter,] <vector expression>)`

or

`<aggr-op>([parameter,] <vector expression>) [without|by (<label list>)]`

`without` removes the listed labels from the result vector, while all other labels are preserved in the output. `by` does the opposite and drops labels that are not listed in the by clause, even if their label values are identical between all elements of the vector.
"
https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators

Reply:
Best course of action here is to use sum without(), with empty without clause. This way sum() don't drop any label.

Reply:
![image](https://github.com/grafana/jsonnet-libs/assets/9431850/c6844045-ff38-4d67-a11e-7b12a9860009)